models/design.js

```diff
@@ -0,0 +1,17 @@
+var mongoose = require("mongoose");
+
+var Schema = mongoose.Schema;
+
+var DesignSchema = new Schema({
```

Comment:
This is great, and we'll re-use your research for the individual react components
1. framework bade
2. Google Font Badge
3. Responsive Badge
...etc.